shell/platform/embedder/tests/embedder_gl_unittests.cc

```diff
@@ -3522,17 +3518,15 @@ TEST_F(EmbedderTest, CompositorRenderTargetsAreInStableOrder) {
 }
 
 TEST_F(EmbedderTest, FrameInfoContainsValidWidthAndHeight) {
-  auto& context = GetEmbedderContext(EmbedderTestContextType::kOpenGLContext);
-
-  EmbedderConfigBuilder builder(context);
-  builder.SetOpenGLRendererConfig(SkISize::Make(600, 1024));
-  builder.SetDartEntrypoint("push_frames_over_and_over");
```

Comment:
Grouped all the context stuff together and builder stuff together, now that context configuration isn't order-dependent with `EmbedderConfigBuilder::Set*RendererConfig()` calls.